packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -99,6 +102,13 @@ const styles = (theme: Theme) => {
         warning: {
             color: theme.palette.warning.main,
         },
+        hasHelperText: {},
```

Comment:
I don't think this is necessary. The presence of a helper text should not influence the field styling, please remove.